src/styles/components/_about_us.scss

```diff
@@ -0,0 +1,5 @@
+.about_us {
+  a {
+    text-decoration: underline;
+  }
+}
```

Comment:
I would apply this css in the _section.scss where we have the rest styles for this page instead of creating a new file

Reply:
@tsvetelinak0 I don't think so because section class is used in the contact us page too: https://github.com/PolyglotDevsLondon/sharespots/blob/develop/templates/core/contact.html#L5

And it might be used in future pages too and they may not want links underlined.

So I'm going to merge this PR as it is.
<hr>
On a side note, in my opinion having a _section css file doesn't seem like a good idea anyway because it is a vague name, and personally I prefer having links underlined by default everywhere but that's a discussion for another day.